packages/devtools_app/test/debugger_screen_test.dart

```diff
@@ -447,54 +447,187 @@ void main() {
       expect(find.text('<async break>'), findsOneWidget);
     });
 
-    testWidgetsWithWindowSize(
-        'Variables shows items', const Size(1000.0, 4000.0),
-        (WidgetTester tester) async {
-      when(debuggerController.variables)
-          .thenReturn(ValueNotifier(testVariables));
-      await pumpDebuggerScreen(tester, debuggerController);
-      expect(find.text('Variables'), findsOneWidget);
+    group('Variables', () {
+      setUp(() {
+        resetRef();
+        resetRoot();
+      });
 
-      final listFinder = find.selectableText('Root 1: _GrowableList (2 items)');
+      testWidgetsWithWindowSize(
+          'Variables shows items', const Size(1000.0, 4000.0),
+          (WidgetTester tester) async {
+        when(debuggerController.variables).thenReturn(
+          ValueNotifier(
+            [
+              buildListVariable(),
+              buildMapVariable(),
+              buildStringVariable('test str'),
+              buildBooleanVariable(true),
+            ],
+          ),
+        );
+        await pumpDebuggerScreen(tester, debuggerController);
+        expect(find.text('Variables'), findsOneWidget);
 
-      // expect a tooltip for the list value
-      expect(
-        find.byTooltip('_GrowableList (2 items)'),
-        findsOneWidget,
-      );
+        final listFinder =
+            find.selectableText('Root 1: _GrowableList (2 items)');
 
-      final mapFinder = find
-          .selectableTextContaining('Root 2: _InternalLinkedHashmap (2 items)');
-      final mapElement1Finder = find.selectableTextContaining("['key1']: 1.0");
-      final mapElement2Finder = find.selectableTextContaining("['key2']: 1.1");
+        // expect a tooltip for the list value
+        expect(
+          find.byTooltip('_GrowableList (2 items)'),
+          findsOneWidget,
+        );
 
-      expect(listFinder, findsOneWidget);
-      expect(mapFinder, findsOneWidget);
-      expect(
-        find.selectableTextContaining("Root 3: 'test str...'"),
-        findsOneWidget,
-      );
-      expect(
-        find.selectableTextContaining('Root 4: true'),
-        findsOneWidget,
-      );
+        final mapFinder = find.selectableTextContaining(
+            'Root 2: _InternalLinkedHashmap (2 items)');
+        final mapElement1Finder =
+            find.selectableTextContaining("['key1']: 1.0");
+        final mapElement2Finder =
+            find.selectableTextContaining("['key2']: 2.0");
+
+        expect(listFinder, findsOneWidget);
+        expect(mapFinder, findsOneWidget);
+        expect(
+          find.selectableTextContaining("Root 3: 'test str...'"),
+          findsOneWidget,
+        );
+        expect(
+          find.selectableTextContaining('Root 4: true'),
+          findsOneWidget,
+        );
+
+        // Expand list.
```

Comment:
move this comment to be above where you expand the list (the tap call). same with expand map below. And add the // Initially ... comments like you have below. Those make the test easy to follow.

Reply:
Done